kubernetes/structure_cron_job.go

```diff
@@ -0,0 +1,59 @@
+package kubernetes
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	batchv2 "k8s.io/client-go/pkg/apis/batch/v2alpha1"
+	"errors"
+)
+
+func flattenCronJobSpec(in batchv2.CronJobSpec) ([]interface{}, error) {
+	att := make(map[string]interface{})
+
+	if in.Schedule != "" {
+		att["schedule"] = in.Schedule
+	} else {
+		return nil, errors.New("You need to define a schedule.")
```

Comment:
Instead of error checking here, could you set `Required = true` on the resource schema in `cronJobSpecFields()`?